src/Audis.Analyzer.Common/Audis.Analyzer.Common/Audis.Analyzer.Common.csproj

```diff
@@ -41,6 +41,7 @@
   </ItemGroup>
```

Comment:
Bitte oben in Zeile 15, 16 und 17 die Version von 5.1 auf 5.2 erhöhen.

Reply:
Ist es Absicht, dass in Zeile 17 bei _Version_ keine 4. Stelle bei der Versionsnummer steht?

Reply:
Ja, weil das die Version für's NuGet Package ist. Dort gibt's nur 3 Stellen.